.travis.yml

```diff
@@ -1,20 +1,20 @@
-# env:
-#   - LARAVEL_VERSION="^6.0" TESTBENCH_VERSION="~4.0" REDIS_DRIVER=phpredis
+env:
+  - LARAVEL_VERSION="^7.0"
```

Comment:
```
env:
  - LARAVEL_VERSION="^6.0" TESTBENCH_VERSION="~4.0"
  - LARAVEL_VERSION="^7.0" TESTBENCH_VERSION="~5.0"
```

Might work with the issues related to https://github.com/stancl/tenancy/pull/304#issuecomment-594857474

Reply:
I think it would work, but I wonder why the current solution doesn't work

Reply:
Seems like this just started working?